src/index.js

```diff
@@ -16,6 +22,23 @@ export default class Server {
                 resolve();
             });
 
+            this.app.get('/download', (req, res) => {
+                res.download('./resources/files/picture.jpg');
+                console.log('123');
+            });
+
+
+            this.app.post('/upload', (req, res) => {
+                console.log('456');
+                const f = req.files.image;
```

Comment:
It's not a common way to name variables. We are trying to avoid using one letter names in production code (of course there some exeptions, e.g. in cases `el.addEventListener('click', e => {})` or `try {} catch(e){}` etc.)
Let's rename `f`-> `imageFile` e.g.